Tiltfile

```diff
@@ -1,31 +1,105 @@
+def printup(*args):
+  if config.tilt_subcommand == "up":
+    print(*args)
+
+script_dir = os.path.join(config.main_dir, 'script')
+
+default_kibana_port = 5601
+kibana_host = "localhost"
+kibana_path = ""
+
+# checks whether (dev mode) kibana is already running at given port,
+# and returns the URL it is redirected to
+def discover_kibana():
```

Comment:
**question**: Is there a reason people would run kibana outside of docker ? 

It feels like this discovery process is adding a lot of "responsibility"/"feature creep" as opposed to a custom path+overlay+docker

Reply:
Yes, during development, where it is not practical to wait for a docker rebuild of Kibana.